docs/standards/nft-2.0/LSP7-Digital-Asset.md

```diff
@@ -5,30 +5,89 @@ sidebar_position: 3
 
 # LSP7 - Digital Asset
 
-:::caution This section is a work in progress.
-:::
-
 :::info Standard Document
 
 [LSP7 - Digital Asset](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-7-DigitalAsset.md)
 
 :::
 
-LSP7 Digital Asset is a standard that aims to describe a fungible asset. The term _fungible_ means that these assets are mutually interchangeable (*e.g., *one token has the same value as another token).
 
-LSP7 Digital Assets are similar to [ERC20](https://eips.ethereum.org/EIPS/eip-20) Tokens.
+## Introduction
+
+Fungible assets represented mainly by **[ERC20](https://eips.ethereum.org/EIPS/eip-20)** and other standards such as **[ERC223](https://eips.ethereum.org/EIPS/eip-223)** and **[ERC777](https://eips.ethereum.org/EIPS/eip-777)** have a lot of limitations in terms of metadata, secure transfers, and asset interaction. This is causing major problems for users seeking, **full control** over which assets they accept or not, and a **simple user experience** while creating, buying, and exchanging assets.
```

Comment:
Might be worth spelling out what these limitations are so we can show by example why LSP standards are an improvement over older standards. We don't really want to say the old standards are bad, but just show that they have limitations over what they can do and how LSPs fix this

```suggestion
Fungible assets represented mainly by **[ERC20](https://eips.ethereum.org/EIPS/eip-20)** and other standards such as **[ERC223](https://eips.ethereum.org/EIPS/eip-223)** and **[ERC777](https://eips.ethereum.org/EIPS/eip-777)** have a lot of limitations in terms of metadata, secure transfers, and asset interaction. This causes problems for users seeking **full control** over which assets they accept and a **simple user experience** while creating, buying, and exchanging assets.
```

Reply:
In the introduction, i try to simplify the problems of the old standards, with few words, so I can detail about the problem in the each relative section